config.tpl.yml

```diff
@@ -28,114 +28,137 @@ APPS:
+    scalers:
+      - type: SqsScaler
+        queues:  [retry-tasks]
+        threshold: 250
 
   - name: notify-delivery-worker-internal
     min_instances: {{ MIN_INSTANCE_COUNT_LOW }}
     max_instances: {{ MAX_INSTANCE_COUNT_LOW }}
-    threshold: 250
-    scalers: [SqsScaler]
-    queues:  [notify-internal-tasks]
+    scalers:
+      - type: SqsScaler
+        queues:  [notify-internal-tasks]
+        threshold: 250
 
   - name: notify-delivery-worker-letters
     min_instances: {{ MIN_INSTANCE_COUNT_LOW }}
     max_instances: {{ MAX_INSTANCE_COUNT_LOW }}
-    threshold: 250
-    scalers: [SqsScaler]
-    queues:  [create-letters-pdf-tasks, letter-tasks]
+    scalers:
+      - type: SqsScaler
+        queues:  [create-letters-pdf-tasks, letter-tasks]
+        threshold: 250
 
   - name: notify-delivery-worker-research
     min_instances: {{ MIN_INSTANCE_COUNT_LOW }}
     max_instances: {{ MAX_INSTANCE_COUNT_LOW }}
-    threshold: 250
-    scalers: [SqsScaler]
-    queues:  [research-mode-tasks]
+    scalers:
+      - type: SqsScaler
+        queues:  [research-mode-tasks]
+        threshold: 250
 
   - name: notify-delivery-worker-priority
     min_instances: {{ MIN_INSTANCE_COUNT_LOW }}
     max_instances: {{ MAX_INSTANCE_COUNT_LOW }}
-    threshold: 250
-    scalers: [SqsScaler]
-    queues:  [priority-tasks]
+    scalers:
+      - type: SqsScaler
+        queues:  [priority-tasks]
+        threshold: 250
 
   - name: notify-delivery-worker-periodic
     min_instances: {{ MIN_INSTANCE_COUNT_LOW }}
     max_instances: {{ MAX_INSTANCE_COUNT_LOW }}
-    threshold: 250
-    scalers: [SqsScaler]
-    queues:  [periodic-tasks, statistics-tasks]
+    scalers:
+      - type: SqsScaler
+        queues:  [periodic-tasks, statistics-tasks]
+        threshold: 250
 
   - name: notify-delivery-worker-receipts
     min_instances: {{ MIN_INSTANCE_COUNT_LOW }}
     max_instances: {{ MAX_INSTANCE_COUNT_HIGH }}
-    threshold: 250
-    scalers: [SqsScaler, ScheduleScaler]
-    queues:  [ses-callbacks]
-    schedule:
-      scale_factor: 0.3
-      workdays:
-        - 08:00-19:00
-      weekends:
-        - 08:00-19:00
+    scalers:
+      - type: SqsScaler
+        queues:  [ses-callbacks]
+        threshold: 250
+      - type: ScheduleScaler
+        schedule:
+          scale_factor: 0.3
+          workdays:
+            - 08:00-19:00
+          weekends:
+            - 08:00-19:00
 
   - name: notify-template-preview
     min_instances: {{ MIN_INSTANCE_COUNT_LOW }}
     max_instances: {{ MAX_INSTANCE_COUNT_TEMPLATE_PREVIEW }}
-    threshold: 8
-    scalers: [SqsScaler, ScheduledJobsScaler, ScheduleScaler]
-    queues:  [create-letters-pdf-tasks, letter-tasks]
-    schedule:
-      scale_factor: 0.4
-      workdays:
-        - 08:00-19:00
-      weekends:
-        - 08:00-19:00
+    scalers:
+      - type: ScheduledJobsScaler
+        threshold: 8
+      - type: SqsScaler
+        queues:  [create-letters-pdf-tasks, letter-tasks]
+        threshold: 8
```

Comment:
8 seems like a weird number here but it was a shared `threshold` between the `ScheduledJobsScaler` and `SqsScaler` - any suggestions for better numbers is welcome

Reply:
It's 8 because preview generation is quite CPU intensive, I don't know if that changed @leohemsted ?

Reply:
don't think it's changed significantly, but i don't know if that `8` was based on actual benchmarking or just a reckon

Reply:
I guess it's the same behaviour as before so call this PR good and change it later? 😁 
(also we have plans to sort out the scaling of template-preview next quarter I think so we can fix these values then)